cadc-rest/src/main/java/ca/nrc/cadc/rest/SyncInput.java

```diff
@@ -239,107 +237,100 @@ public String getParameter(String name)
             }
-        }
-        else
-        {
+        } else {
             String contentType = request.getContentType();
-            if (contentType != null)
-            {
+            if (contentType != null) {
                 int i = contentType.indexOf(';');
-                if (i > 0)
+                if (i > 0) {
                     contentType = contentType.substring(0, i);
+                }
             }
             log.debug("Content-Type: " + contentType);
-            if (contentType != null && contentType.equalsIgnoreCase(RestAction.URLENCODED))
-            {
+            if (contentType != null && contentType.equalsIgnoreCase(RestAction.URLENCODED)) {
                 Enumeration<String> names = request.getParameterNames();
-                while (names.hasMoreElements())
-                {
+                while (names.hasMoreElements()) {
                     String name = names.nextElement();
                     processParameter(name, request.getParameterValues(name));
                 }
-            }
-            else if (contentType != null && contentType.startsWith(RestAction.MULTIPART))
-            {
-            	try
-            	{
+            } else if (contentType != null && contentType.startsWith(RestAction.MULTIPART)) {
+                try {
                     ServletFileUpload upload = new ServletFileUpload();
                     FileItemIterator itemIterator = upload.getItemIterator(request);
                     processMultiPart(itemIterator);
-            	}
-            	catch(FileUploadException ex)
-            	{
+                } catch (FileUploadException ex) {
                     throw new IOException("Failed to process " + RestAction.MULTIPART, ex);
-            	}
-            }
-            else
-            {
+                }
+            } else {
                 processStream(null, contentType, request.getInputStream());
             }
         }
     }
 
-    private void processParameter(String name, String[] values)
-    {
-    	this.params.put(name, Arrays.asList(values));
+    private void processParameter(String name, String[] values) {
+        this.params.put(name, Arrays.asList(values));
     }
 
     private void processMultiPart(FileItemIterator itemIterator)
-        throws FileUploadException, IOException, ResourceNotFoundException
-    {
-        while (itemIterator.hasNext())
-        {
+        throws FileUploadException, IOException, ResourceNotFoundException {
+        while (itemIterator.hasNext()) {
             FileItemStream item = itemIterator.next();
             String name = item.getFieldName();
             InputStream stream = item.openStream();
-            if (item.isFormField())
-                processParameter(name, new String[] { Streams.asString(stream) });
-            else
+            if (item.isFormField()) {
+                final List<String> existingItems = this.params.get(name);
```

Comment:
Why not just add the new value to the existing list? You could just have processParameter do this much more simply by getting the current list from the params map, create and put if null, and add the value(s).